server/views/partials/header.ejs

```diff
@@ -24,11 +24,12 @@
         <li class="nav-item">
           <a class="nav-link" href="/surveyavailable"><i class="fas fa-gas-pump fa-lg"></i> Survey Available</a>
         </li>
-
         <li class="nav-item">
           <a class="nav-link" href="/login"><i class="fas fa-id-card fa-lg"></i> Login</a>
         </li>
-
+         <li class="nav-item">
+          <a class="nav-link" href="/logout"><i class="fas fa-id-card fa-lg"></i> Logout</a>
```

Comment:
I believe there should be a check for if the user is logged in or not to show the login and logout links? Did you talk with the web designer to take care of this part?